lib/abide_dev_utils/errors/xccdf.rb

```diff
@@ -20,5 +20,9 @@ class ControlPartsError < GenericError
     class ProfilePartsError < GenericError
       @default = 'Failed to extract parts from profile name:'
     end
+
+    class CliOptionsConflict < GenericError
```

Comment:
This is a great new error to add, but should probably live in a different file because it's not specific to the XCCDF stuff.